### DISCUSSION TITLE
Negative budgets are converted to positive by Firefly III

### DISCUSSION THREAD

**Discussion by PsychoGame** [firefly-iii/firefly-iii#7373] [Firefly III]
About an hour ago I've submitted a bug request for a regression which occurred between the transition of 5 to 6.
This issue has been closed with comment that this was never intended.
I have noticed in the bug report that it was never intended to be possible, but what are the considerations?
When we had overspend budgets in the previous month, and the new budget addition still left the category overspend we had a negative budget category where we wanted to pay more attention to in regards of spenditure.

**Comment by Zyadon7** [comment]
I agree.  I miss this feature and relied on it for any budget that overspent.  Int eh current version .20 a negative balance gets converted to 1 which isn't consistent for amounts owed.
